fdbclient/ManagementAPI.actor.cpp

```diff
@@ -3215,6 +3215,91 @@ ACTOR Future<size_t> getBulkDumpCompleteTaskCount(Database cx, KeyRange rangeToR
 	return completeTaskCount;
 }
 
+ACTOR Future<Optional<BulkDumpRestoreState>> getOngoingBulkDumpRestoreJob(Database cx) {
+	state RangeResult rangeResult;
+	state Transaction tr(cx);
+	loop {
+		try {
+			// At most one job at a time, so looking at the first returned range is sufficient
```

Comment:
Can I do a bulkload while a bulkdump is going on or is it exclusively one of these operations only?